boat-scaffold/src/main/templates/boat-angular/api.module.mustache

```diff
@@ -2,10 +2,6 @@ import { NgModule, ModuleWithProviders, SkipSelf, Optional } from '@angular/core
 import { {{configurationClassName}}{{#withMocks}}, CONFIG_TOKEN{{/withMocks}} } from './configuration';
 import { HttpClient } from '@angular/common/http';
 
-{{#withMocks}}
-import { DataModulesManager } from "@backbase/foundation-ang/data-http";
```

Comment:
We also need to remove the other references to this in this file.

Reply:
There's also [this section](https://github.com/Backbase/backbase-openapi-tools/blob/8875bcfd9aecaf81b16e680ea7a8b76b8273782c/boat-scaffold/src/main/templates/boat-angular/api.module.mustache#L34-L42) to remove further down